ui/src/App.vue

```diff
@@ -1,48 +1,44 @@
 <template>
-  <div class="container">
-    <SearchBar @termSubmit="onSubmit"></SearchBar>
-    <div class="row">
-      <CardListItem></CardListItem>
-    </div>
-
+  <div class="ui container">
+    <h2 class="ui center aligned header">Job Hooroom</h2>
+      <SearchBar />
+      <div v-if="this.graphData" class="flex-box">
+        <GraphCard v-for="(value, index) in this.graphData.data" :key="index" :graphProp="value.buckets" :title="index" />
```

Comment:
After updating the getter, you will need to update here as well
```suggestion
        <GraphCard v-for="(value, index) in this.graphData" :key="index" :graphProp="value.buckets" :title="index" />
```